### PR TITLE
Increase max_accelerated_files.

### PR DESCRIPTION
**Pull request by singularo**
Starting a MR to pull in these performance changes.

See SAPP-1201

### CODE REVIEW

File: files/php_custom.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 opcache.memory_consumption = 128
 opcache.interned_strings_buffer = 8
-opcache.max_accelerated_files = 4000
+opcache.max_accelerated_files = 30000
```
Comment (pingers) [inline files/php_custom.ini:5]: @dpi I heard from Adam there might be some more things we should tweak. Any thoughts?

**Comment by dpi** [comment]
Hey @pingers, identified below:

_values come from other images, significantly higher than existing images:_

```ini
apc.shm_size = 128M
apc.ttl = 86400
opcache.max_accelerated_files = 40000
```

And the following for dev image, on top of above.

```ini
; arguably we can skip it for CLI apps:
opcache.enable_cli = 0
; this will accelerate dev, as the current setup is 60. it can sometime take a VERY long time for file updates to bubble to the app. This avoids needs to wait/cr/completely change the file - to trigger an update.
opcache.revalidate_freq = 2
; arguably not necessary on dev
apc.slam_defense = 0 
```

**Comment by singularo** [comment]
Left opcache.max_accelerated_files = 30000 as we have 15k in the sites I checked, so still plenty of headroom.

**Review by pingers — APPROVED** [review]
:shipit:

**Review by dpi — APPROVED** [review]

**Pull request merged**